sdcm/stress/latte_thread.py

```diff
@@ -122,8 +122,23 @@ def build_stress_cmd(self, cmd_runner):
                            '--ssl-cert /etc/scylla/ssl_conf/client/test.crt '
                            '--ssl-key /etc/scylla/ssl_conf/client/test.key')
 
+        custom_schema_params = ""
+        if latte_schema_parameters := self.params['latte_schema_parameters']:
+            # NOTE: string parameters in latte must be wrapped into escaped double-quotes: foo="\"bar\""
+            for k, v in latte_schema_parameters.items():
+                processed_v = v
+                try:
+                    processed_v = int(v)
+                except Exception:  # pylint: disable=broad-except  # noqa: BLE001
+                    if v not in ('true', 'false'):
+                        processed_v = r"\"%s\"" % v
+                custom_schema_params += " -P {k}={v}".format(k=k, v=processed_v)
         cmd_runner.run(
+<<<<<<< HEAD
             cmd=f'latte schema {script_name} {ssl_config} -- {hosts}',
+=======
+            cmd=f'latte schema {script_name} {ssl_config} {auth_config}{custom_schema_params} -- {hosts}',
```

Comment:
seems like we are missing the auth configuration in this branch

Reply:
We also don't CI jobs configuration.
So, latte is not used in this branch.